### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Comparison_AI4D: Name of the elements under "Source name and include" is missing.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6796]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
Additional Details : AI4D, MAS4.1.2, MAS1.3.1, AllyAuto

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Open AI4D and run the fast pass for the table.
13. Check whether name of the elements is present or not.

**Actual:**
Name of the elements under "Source name and include" is missing.

**Expected:**
Name should be provided for the elements which is present under "Source name and Include".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know the information about the control name.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
![6796_A11y_AzureDataStudio(Schema,Dacpac,Backup)_Comparison_AI4D](https://user-images.githubusercontent.com/24580570/63321430-53a19d80-c33e-11e9-90e3-34496fcb2474.png)

**Comment by udeeshagautam** [comment]
Added to more info questions

**Comment by udeeshagautam** [comment]
Question on this - the check boxes will be fixed but for the other column there is no name because name = data in the column. and there is no data in those cells as a result of comparison. Do we still need a name ?

**Comment by fsteffi** [comment]
Issue is partially fixed For the check boxes name is still missing , for more information You can refer the attachment.
![6796_Reopened](https://user-images.githubusercontent.com/24580570/66647549-d3fdc400-ec46-11e9-8ad5-de365e072ebe.png)

**Comment by udeeshagautam** [comment]
@fsteffi we have added the aria-label for the check boxes -which works are name (for all controls) and gets the narrartion corrected too. We don't explicitly have acess to name control - can you suggest if  there is a way to add name explicitly. 
Also, since the narration is correct can you help understand if there is still user impact because of this behavior?

**Comment by fsteffi** [comment]
@udeeshagautam Yes it is fine to provide Aria label, Screenreader reads the checkbox along with the header.
Hence closing this bug.
![6796_Closed](https://user-images.githubusercontent.com/24580570/66746523-a8c1e180-ee9f-11e9-8aab-a6d1b58a6327.png)
